io.github.jotd666.gods-deluxe.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.jotd666.gods-deluxe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: gods-deluxe.sh
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+cleanup:
+  - /include
+  - '*.a'
+  - '*.la'
+  - /lib/pkgconfig
+  - /share/man
+  - /share/cmake
+modules:
+
+  # Install Java
+  - name: install-openjdk
+    buildsystem : simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  # Install gods-deluxe from package
+  - name: gods-deluxe-build
```

Comment:
Is this x86-64 only? You have a bunch of x86_64 shared libraries in `/app/jre`. If aarch64 is not supported add a flathub.json